tests/Unit/Domain/CoordinateMaps/TimeDependent/Test_RotScaleTrans.cpp

```diff
@@ -94,70 +94,85 @@ void test_RotScaleTrans() {
   const std::pair<std::string, std::string> scale_pair{"expansion_a",
                                                        "expansion_b"};
   // Rotation, Scaling, Translation Non-Rigid
-  const domain::CoordinateMaps::TimeDependent::RotScaleTrans<Dim>
+  domain::CoordinateMaps::TimeDependent::RotScaleTrans<Dim>
       rot_scale_trans_map_non_rigid{scale_pair,    "rotation_angle",
                                     "translation", inner_radius,
                                     outer_radius,  false};
+  rot_scale_trans_map_non_rigid =
```

Comment:
Instead of overwriting the variable, just serialize/deserialize immediately:

```cpp
  const domain::CoordinateMaps::TimeDependent::RotScaleTrans<Dim>
     serialize_and_deserialize(rot_scale_trans_map_non_rigid{scale_pair,    "rotation_angle",
                                    "translation", inner_radius,
                                    outer_radius,  false});
```
Same for the rest as well.